src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -15049,28 +15049,27 @@ namespace SurfaceGeometry {
                     }
                     if (solarSetup && constr.TypeIsAirBoundaryMixing) {
                         // Set up mixing air boundaries only once, during solar setup
-                        int zoneNum1 = min(surf.Zone, state.dataSurface->Surface(surf.ExtBoundCond).Zone);
-                        int zoneNum2 = min(surf.Zone, state.dataSurface->Surface(surf.ExtBoundCond).Zone);
+                        int spaceNum1 = min(surf.spaceNum, state.dataSurface->Surface(surf.ExtBoundCond).spaceNum);
+                        int spaceNum2 = max(surf.spaceNum, state.dataSurface->Surface(surf.ExtBoundCond).spaceNum);
```

Comment:
Clean up air boundary surface mixing object tracking. Stores some info here, and then creates mixing instances later in HeatBalanceAirManager.
This also fixes the duplicate mixing instances. Note the old code resulted in the same value for both zoneNum1 and zoneNum2 making the duplicate check useless.